### PR TITLE
[securemsg] Add clear command to clear message queue without action

### PR DESCRIPTION
**Pull request by vague666**

### CODE REVIEW

File: scripts/securemsg.pl [hunk 1]
```diff
@@ -516,6 +517,38 @@ sub cmd_show {
     }
 }
 
+sub cmd_clear {
+    my ($args, $server, $witem) = @_;
+    my ($nick, $net) = _get_nn($args);
+    my $server;
+
+    if((!defined $nick) || (!exists $messages{$nick})) {
+        usage;
```
Comment (bw1) [inline scripts/securemsg.pl:526]: 175 `print("Usage: sm add|del <nick>|<host> list of nicks/hosts | sm nicks|hosts | sm accept|reject <nick> [net <chatnet>] [message] | sm show <nick> [net <chatnet>] | sm help");`

Comment (vague666) [reply scripts/securemsg.pl:526]: Are you saying usage is printed if you try to use /sm clear <nick> or that usage is missing info on clear?

Comment (bw1) [reply scripts/securemsg.pl:526]: `sub usage` miss the info about clear

Comment (vague666) [reply scripts/securemsg.pl:526]: Thanks! I updated the help but missed that bit :)

**Comment by bw1** [comment]
```
588     Irssi::settings_add_str('SecureMsg', $types{$_}, '');
591 Irssi::settings_add_str('Securemsg', 'securemsg_customhold', '');
592 Irssi::settings_add_str('Securemsg', 'securemsg_customaccept', '');
593 Irssi::settings_add_str('Securemsg', 'securemsg_ignoretime', '');
```
see camelCase

**Comment by bw1** [comment]
see also help on top of file

**Comment by bw1** [comment]
merge or squash?

**Pull request merged**